src/ui/pages/BridgeForm/ReverseButton/styles.module.css

```diff
@@ -0,0 +1,44 @@
+@keyframes spin {
+  from {
+    transform: perspective(400px) rotate3d(0, 1, 0, 90deg);
+    animation-timing-function: ease-in;
+    opacity: 1;
+  }
+  40% {
+    transform: perspective(400px) rotate3d(0, 1, 0, -20deg);
+    animation-timing-function: ease-in;
+  }
+  60% {
+    transform: perspective(400px) rotate3d(0, 1, 0, 10deg);
+    opacity: 1;
+  }
+  80% {
+    transform: perspective(400px) rotate3d(0, 1, 0, -5deg);
+  }
+  to {
+    transform: perspective(400px);
+  }
+}
+
+.reverseButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  border: 3px solid var(--neutral-100);
+}
+
+.spinArrow {
+  transform-origin: 50% 50%;
+  color: var(--neutral-500);
+}
+
+.reverseButton:focus .spinArrow {
```

Comment:
This style triggers animation on focus, not on click. Which works with click but not with keyboard navigation

Reply:
thanks, I copied this part from webapp as is :)
I just removed `:focus` for now